src/components/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { Status } from '../types/Status';
+
+interface Props {
+  activeTodosCount: number;
+  selectedStatus: Status;
+  setSelectedStatus: (status: Status) => void;
+  deleteAllCompleted: () => void;
+  clearAllVisible: boolean;
```

Comment:
[optional fix]

Common convention is start names for `boolean` from "is", "are", "can", etc

```suggestion
  canClearAllVisible: boolean;
```